packages/lib/src/toolbar/controls/ToggleGroup.tsx

```diff
@@ -24,16 +29,17 @@ function useToggleGroupProps(): ToggleGroupProps {
   return context;
 }
 
-interface BtnProps {
+interface BtnProps extends HTMLAttributes<HTMLButtonElement> {
```

Comment:
The `extends` is not needed since we do not propagate `props` to `button`, right ?